server/src/main/kotlin/com/xebia/functional/xef/server/http/routes/providers/Routes.kt

```diff
@@ -0,0 +1,16 @@
+package com.xebia.functional.xef.server.http.routes.providers
+
+import com.xebia.functional.xef.server.http.routes.providers.openai.oaiFiles
+import com.xebia.functional.xef.server.http.routes.providers.openai.oaiFineTuning
+import io.ktor.client.*
+import io.ktor.server.auth.*
+import io.ktor.server.routing.*
+
+fun Routing.oaiRoutes(
```

Comment:
If this is the `oaiRoutes`, wouldn't chat completion and embeddings be part of these? I think we have some confusion regarding how routes are organized in the Xef server.

Reply:
Yes, @Montagon had to curb me, not too change too much in one PR again. The other endpoints, if approved, will have to get integrated with the my new structure. This includes the forwarding logic and registering the endpoints as here.

Reply:
I said this comment to you referred to when you have the other PR in this one.

Reply:
The same applies here, no? You made some changes in the existing endpoints and files. I didn't want to mess (or get in conflict) with that. You instructed me to not change the existing endpoints at that point, me acuerdo.